build.gradle

```diff
@@ -255,6 +281,62 @@ project('segmentstore:server:host') {
     }
 }
 
+
+project('segmentstore:server:hostwithbindings') {
```

Comment:
This creates a new segmentstore binary distribution is which contains the jar files for custom bindings. This will be used for building the docker container. This will ensure that the docker container will work out of the box with custom bindings without any special action from the deployment.